### PR TITLE
Default build number to 0 in version

### LINKED ISSUE #2324: Build number not set in ISO version on local build
Build number not defaulted to 0 in `isos/base/utils.sh`

### PR DESCRIPTION
**Pull request by andrewtchin**
Fixes #2324 
@gigawhitlocks @jzt this fixes `vic-machine create` version issues you were seeing

### CODE REVIEW

File: pkg/version/version.go [hunk 1]
```diff
@@ -55,11 +55,17 @@ func String() string {
 }
 
 func GetBuild() *Build {
+	var b string
+	if BuildNumber == "" {
+		b = "0"
```
Comment (caglar10ur) [inline pkg/version/version.go]: Just set BuildNumber to "0", no need to have another variable or an else statement

Comment (gigawhitlocks) [inline pkg/version/version.go]: Also, now that you point this out, where does BuildNumber get defined? It looks like this function should be a method w/ BuildNumber a receiver member.

Comment (andrewtchin) [inline pkg/version/version.go]: good point BuildNumber is set here: https://github.com/vmware/vic/blob/master/Makefile#L57 based on env var which usually comes from drone

**Comment by jzt** [comment]
LGTM

**Comment by andrewtchin** [comment]
♻️

**Comment by andrewtchin** [comment]
♻️

**Comment by andrewtchin** [comment]
lgtm

**Pull request merged**